biosynseq/generate.py

```diff
@@ -40,14 +40,12 @@ def generate_fasta(
     Dict[str, List[str]]
         Unique generated sequences.
     """
-    # obtain model
-    model = model_strategy.get_model()
-    model.cuda()
-    # generate non-redundant sequences
+    # Obtain model
+    model = model_strategy.get_model(DNATransformer).cuda()
```

Comment:
Do we want to make the cuda call an option here? I think it's fine hard coded but something to consider :)